common/FileSystem.cpp

```diff
@@ -1264,6 +1265,71 @@ size_t FileSystem::ReadFileWithPartialProgress(std::FILE* fp, void* dst, size_t
 	return done;
 }
 
+#ifdef _WIN32
+static std::span<const u8> MapBinaryFileForRead(HANDLE handle)
+{
+	LARGE_INTEGER size;
+	if (!GetFileSizeEx(handle, &size) || size.QuadPart == 0)
+		return {};
+	HANDLE mapping = CreateFileMappingA(handle, nullptr, PAGE_READONLY, size.HighPart, size.LowPart, nullptr);
```

Comment:
What is the reason for using the ansi version over `CreateFileMappingW`?
It looks like doesn’t really matter which here as we aren’t passing a string to it.